### PR TITLE
Api method

### PR DESCRIPTION
**Pull request by dchess**
Replaces the previous method of using tab cmd in Docker with a the Tableau server client library leveraging the rest API for improved speed. 

Will include updates to the Google Drive uploader and mailer on a separate PR.

### CODE REVIEW

File: students.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+SELECT TOP 10
```
Comment (susanxiong) [inline students.sql:1]: remove `TOP 10`?

File: students_grade.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+SELECT TOP 10
```
Comment (susanxiong) [inline students_grade.sql:1]: remove `TOP 10`?

File: students.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      lks.SchoolName_Tableau AS SchoolName
+    , cs.CurrGradeLevel
+    , cs.student_number
+    , cs.lastfirst
+FROM custom.vw_enrollment_course_section_ps cs
+LEFT JOIN custom.lkSchools lks
+    ON cs.schoolid_record = lks.SystemSchoolID
+WHERE cs.Schoolyear4digit = 2021
+-- WHERE cs.Schoolyear4digit = custom.fn_SchoolYear4Digit (GETDATE())
```
Comment (susanxiong) [inline students.sql:10]: remove comment or replace hard coded line

File: students_grade.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      lks.SchoolName_Tableau AS SchoolName
+    , cs.CurrGradeLevel
+    , cs.student_number
+    , cs.lastfirst
+FROM custom.vw_enrollment_course_section_ps cs
+LEFT JOIN custom.lkSchools lks
+    ON cs.schoolid_record = lks.SystemSchoolID
+WHERE cs.Schoolyear4digit = 2021
+-- WHERE cs.Schoolyear4digit = custom.fn_SchoolYear4Digit (GETDATE())
```
Comment (susanxiong) [inline students_grade.sql:10]: remove comment or replace hard coded line

**Review by susanxiong — COMMENTED** [review]
Overall looks good to me. I had small questions about the SQL queries.

Also, I'm not seeing drive.py or pydrive being used anywhere. Is it being used? How are these usually shared with schools once we print them?

**Comment by dchess** [comment]
@susanxiong Good catch on the SQL. Those were in there for testing. Due to the length of time to run the job doing a TOP N query limits the time spent watching it. The schoolyear4digit hardcode is also necessary during the inter-summer period.

Pydrive isn't being used in this PR. I will submit a separate one with an update to that logic as well as the tje mailgun update.

**Review by susanxiong — APPROVED** [review]

**Pull request merged**